docs/source/examples/plot_toll_analysis.py

```diff
@@ -0,0 +1,138 @@
+"""
+Toll analysis
+=============
+
+On this example we show how to perform traffic assignment considering fixed
+link costs (e.g. tolls) besides travel time. We also do assignment with
+three different vehicle classes
+"""
+
+## Imports
+from tempfile import gettempdir
+from os.path import join
+import urllib
+import zipfile
+
+# %%
+from aequilibrae import logger, Project
+from aequilibrae.paths import TrafficAssignment, TrafficClass
+import logging
+import sys
+
+# We the project open, we can tell the logger to direct all messages to the terminal as well
+stdout_handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter("%(asctime)s;%(name)s;%(levelname)s ; %(message)s")
+stdout_handler.setFormatter(formatter)
+logger.addHandler(stdout_handler)
+
+# %%
+
+disk_pth = join(gettempdir(), 'sioux_falls_multi-Class')
+urllib.request.urlretrieve('https://www.aequilibrae.com/data/sioux_falls_multi-Class.zip', disk_pth + '.zip')
+
+with zipfile.ZipFile(disk_pth + '.zip') as zf:
+    zf.extractall(gettempdir())
+
+# %%
+project = Project()
+project.open(disk_pth)
+
+# %%
+proj_matrices = project.matrices
+proj_matrices.update_database()
+mat_list = proj_matrices.list()
+
+# %%
+# we build all graphs
+project.network.build_graphs()
+carGraph = project.network.graphs['c']
+truckGraph = project.network.graphs['T']
+motoGraph = project.network.graphs['M']
+
+# %%%
+
+matrix_name = 'demand_mc_omx'
+carDemand = proj_matrices.get_matrix(matrix_name)
+carDemand.computational_view('car')
+carClass = TrafficClass(carGraph, carDemand)
+carClass.set_pce(1)
+carClass.set_vot(35)
+carClass.set_fixed_cost('toll', 0.025)
+
+# %%
+
+
+motoDemand = proj_matrices.get_matrix(matrix_name)
+motoDemand.computational_view('motorcycle')
+motoClass = TrafficClass(motoGraph, motoDemand)
+motoClass.set_pce(0.2)
+motoClass.set_vot(35)
+# Fixed cost can be any field (or field_AB/BA in the network).  And the factor defaults to 1.0
+motoClass.set_fixed_cost('toll', 0.0125)
+
+# %%
+truckDemand = proj_matrices.get_matrix(matrix_name)
+truckDemand.computational_view('trucks')
+truckClass = TrafficClass(truckGraph, truckDemand)
+truckClass.set_pce(1.5)
+truckClass.set_vot(35)
+truckClass.set_fixed_cost('toll', 0.05)
+# %%
+
+
+assig = TrafficAssignment()
+# Different (and better) than some commercial software, your results are as proportional as the
```

Comment:
do we want to remove the first part of the sentence?

Reply:
Not really....  :-)

Reply:
Decided to change. You were right, @janzill